makepkg.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import glob
+import os
+import os.path
+import shutil
+import subprocess
+import sys
+
+from azurelinuxagent.common.version import AGENT_NAME, AGENT_VERSION, AGENT_LONG_VERSION
+from azurelinuxagent.ga.update import AGENT_MANIFEST_FILE
+
+
+MANIFEST = '''{{
+    "name": "{0}",
+    "version": 1.0,
+    "handlerManifest": {{
+        "installCommand": "",
+        "uninstallCommand": "",
+        "updateCommand": "",
+        "enableCommand": "{1}",
+        "disableCommand": "",
+        "rebootAfterInstall": False,
+        "reportHeartbeat": False
+    }}
+}}'''
+
+ENABLE_CMD = "python -c 'from azurelinuxagent import agent; agent.main([\"-run-exthandlers\"]);' {0}"
```

Comment:
@brendandixon I'm just not sure if we're missing any preliminary setup (like logging to certain files or some other stuff) by invoking `agent.main` directly instead of having the args parser run and pick up `-run-exthandlers` etc.

If you're confident that these two methods of launching it has no difference, this works for me (worst case, we can always auto-update) otherwise we should probably try to make that syntax work... It'd be cleaner.